### PR TITLE
ci: re-enable adev tests

### PR DESCRIPTION
**Pull request by JeanMeche**

### CODE REVIEW

File: adev/src/app/features/references/api-reference-details-page/api-reference-details-page.component.spec.ts [hunk 1]
```diff
@@ -68,15 +68,17 @@ describe('ApiReferenceDetailsPage', () => {
     expect(component).toBeTruthy();
   });
 
-  it('should render tabs for all elements with tab attribute', async () => {
+  // TODO: fix & re-enable
```
Comment (devversion) [inline adev/src/app/features/references/api-reference-details-page/api-reference-details-page.component.spec.ts:71]: Nit: do we have an issue for tracking? 😄

**Review by devversion — APPROVED** [review]

File: adev/src/app/features/references/api-reference-details-page/api-reference-details-page.component.spec.ts (hunk 1 above)
Comment (devversion) [reply adev/src/app/features/references/api-reference-details-page/api-reference-details-page.component.spec.ts:71]: (or should we have one?)

Comment (JeanMeche) [reply adev/src/app/features/references/api-reference-details-page/api-reference-details-page.component.spec.ts]: I went ahead and fixed the tests !

**Comment by JeanMeche** [comment]
Yeah let's go with it if the CI is fine !

**Comment by thePunderWoman** [comment]
This PR was merged into the repository by commit 7263917793ee57bc83dad148ea109dd007e9bd4d.

The changes were merged into the following branches: main, 19.0.x

**Comment by thePunderWoman** [comment]
This seems to have caused real adev test failures at head. I think we need to revert it.

**Comment by thePunderWoman** [comment]
Revert is in https://github.com/angular/angular/pull/58592

**Review by thePunderWoman — APPROVED** [review]

**Comment by thePunderWoman** [comment]
This PR was merged into the repository by commit 750432337b6df26ddb1497b9d03f1a29f723166e.

The changes were merged into the following branches: main, 19.0.x

**Pull request closed without merging**